js/apps/viewer/src/app/animation-player/animation-player.tsx

```diff
@@ -25,6 +25,7 @@ import {onKeyboardLockChange, onPageVisibilityChange} from '../../common/events'
 import Controls from './controls';
 import ProgressBar from './progress-bar';
 import {frameToTimeStr} from './utils';
+import AnimationSpeedController from './animation-speed-controller';
```

Comment:
Move this above the `Controls` import (run `yarn test` at the root of the `js` dir to catch these format issues, I must have mentioned it before).